src/emc/usr_intf/axis/scripts/axis.py

```diff
@@ -3340,12 +3350,17 @@ def units(s, d=1.0):
 
 kinsmodule=inifile.find("KINS", "KINEMATICS").lower()
 kins_is_trivkins = False
+trivkinstype = "none"
 if kinsmodule.split()[0] == "trivkins":
     kins_is_trivkins = True
     trivkinscoords = "XYZABCUVW"
     for item in kinsmodule.split():
         if "coordinates=" in item:
             trivkinscoords = item.split("=")[1]
+        if "kinstype=" in item:
+            trivkinstype = item.split("=")[1]
+        else:
+            trivkinstype = "single"
```

Comment:
Suppose that 'kinsmodule' is something like `trivkins kinstype=BOTH coordinates=xyzw`.  On the second iteration of the loop `trivkinstype` will be set to `BOTH`, but on the third iteration it will be set back to `single`.  Please revise the pull request so that this problem is fixed, or tell me if I'm overlooking something.